### ISSUE TITLE
Content is not going to be scraped

### ISSUE THREAD

**Issue by orangecoding** [lewisdonovan/google-news-scraper#53]
Hey,

when I try to scrape the content, the content itself is always null. Seems like puppeteer is trying to parse the google news page rather than the actual news page..

Steps to reproduce:
```
import googleNewsScraper from 'google-news-scraper';

const articles = await googleNewsScraper({
    timeframe: '12h',
    getArticleContent: true,
    baseUrl: 'https://news.google.com/topics/CAAqJggKIiBDQkFTRWdvSUwyMHZNRGRqTVhZU0FtVnVHZ0pKVGlnQVAB',
    queryVars: {
        gl:"DE"
    }
});

console.log(articles)
```

**Comment by lewisdonovan** [comment]
Looks like an issue with `prettyURLs`. They may have changed the way these are formatted, I'm working on a fix now. 

Running with `prettyURLs: false` seems to work fine though.

**Comment by lewisdonovan** [comment]
Fixed in [0deb4d6](https://github.com/lewisdonovan/google-news-scraper/commit/0deb4d647b918e9bdc6b8ed486d1662a5ba429ae)

**Comment by orangecoding** [comment]
@lewisdonovan Well.. now I get
```
Uncaught TypeError: getPrettyUrl is not a function. index.js:107
```

**Comment by lewisdonovan** [comment]
Fixed in [2112a92](https://github.com/lewisdonovan/google-news-scraper/commit/2112a9292a35c1af4118408ee9c4297740fba765)
